ocs_ci/framework/pytest_customization/marks.py

```diff
@@ -392,6 +393,18 @@
     ),
     reason="Test runs ONLY on Fusion HCI provider and client clusters",
 )
+# when run_on_all_clients marker is used, there needs to be added cluster_index
+# parameter to the test to prevent any issues with the test parametrization
+run_on_all_clients = pytest.mark.run_on_all_clients
+try:
+    client_indexes = [
+        pytest.param(*[idx]) for idx in config.get_consumer_indexes_list()
+    ]
+    run_on_all_clients = pytest.mark.parametrize(
+        argnames=["cluster_index"], argvalues=client_indexes, indirect=True
+    )
+except ClusterNotFoundException:
```

Comment:
just curious - on what conditions would we end up in this exception in a test flow?